helix-core/src/test/java/org/apache/helix/integration/rebalancer/WagedRebalancer/TestWagedRebalance.java

```diff
@@ -383,20 +383,13 @@ public void testMaxPartitionLimitation() throws Exception {
       configAccessor.setClusterConfig(CLUSTER_NAME, clusterConfig);
       Thread.sleep(300);
```

Comment:
Can we use TestHelper.verify()? Why are we sleeping here?

Reply:
That won't work. This wait is to ensure the controller finishes calculating. Till now, we don't have a good way to wait. If we use the verifier and the it checks before the calculating finish, the verify method would return immediately and prematurely.